### PR TITLE
Migration update

### PR DESCRIPTION
**Pull request by dkulyk**
Fix quote default value(use scheme);
Add unsigned contains in ColumnSchemaBuilder(type control by scheme);

### CODE REVIEW

File: framework/db/ColumnSchemaBuilder.php [hunk 1]
```diff
@@ -83,6 +93,16 @@ public function unique()
     }
 
     /**
+     * Adds a `UNSIGNED` constraint to the column.
+     * @return $this
+     */
+    public function unsigned()
+    {
+        $this->isUnique = true;
```
Comment (Renkas) [inline framework/db/ColumnSchemaBuilder.php]: Shouldn't this be:
$this->isUnsigned = true;

Comment (samdark) [inline framework/db/ColumnSchemaBuilder.php]: Definitely.

File: framework/db/ColumnSchemaBuilder.php [hunk 2]
```diff
@@ -46,17 +50,23 @@ class ColumnSchemaBuilder extends Object
      * @var mixed default value of the column.
      */
     protected $default;
+    /**
+     * @var Schema $schema Used schema
+     */
+    protected $schema;
```
Comment (cebe) [inline framework/db/ColumnSchemaBuilder.php]: schema should not be coupled to this class. we already have DBMS specific implementations of ColumnSchemaBuilder.

**Comment by SilverFire** [comment]
I don't think that this PR can be merged as is.

**Pull request closed without merging**